index.html

```diff
@@ -7,25 +7,31 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link rel="stylesheet" href="style.css" />
   <link rel="preconnect" href="https://fonts.googleapis.com" />
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link rel="xpreconnect" href="https://fonts.gstatic.com" crossorigin />
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet" />
+  <script defer src="script.js"></script>
   <title>Salomon - Portfolio</title>
 </head>
 
 <body>
   <header id="page-header">
     <a href="#" id="my-logo" class="blue-text">My logo</a>
-    <nav>
-      <img src="icons/Hamburger-menu.png" id="hamburger-menu"  alt="menu"/>
-      <ul id="header-ul">
+    <nav id="page-nav">
+      <img src="icons/Hamburger-menu.png" id="hamburger-menu" alt="menu" />
+      <ul id="header-ul" class="header-ul">
+        <li class="header-li exit-li">
+          <button class="nav-exit-button">
+            <img src="icons/exit.png" alt="exit-button">
```

Comment:
- [ ] OPTIONAL
 Kindly ensure that the _hamburger_ and the _close_ button are in the same position when toggled.

- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ behave as buttons when the mouse hovers over them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events.